src/features/Tiles/PeopleTiles/styled.js

```diff
@@ -46,6 +46,7 @@ export const PersonTile = styled(Link)`
     background-color: ${({ theme }) => theme.colors.tile.background};
     box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
     border-radius: 5px;
+    word-break: break-word;
```

Comment:
I find this addition negative. In my opinion CSS works better with default setting